darts/dataprocessing/dtw/cost_matrix.py

```diff
@@ -0,0 +1,119 @@
+from typing import Tuple, Dict
+import numpy as np
+
+from .window import Window, CRWindow
+from abc import abstractmethod
+import array
+from itertools import repeat
+
+Elem = Tuple[int, int]
+
+class CostMatrix:
```

Comment:
I think it might sense to inherit from abc.ABC here (https://docs.python.org/3/library/abc.html)